.update/version

```diff
@@ -1,7 +1,7 @@
 # Available DietPi version
 G_REMOTE_VERSION_CORE=7
-G_REMOTE_VERSION_SUB=1
-G_REMOTE_VERSION_RC=2
+G_REMOTE_VERSION_SUB=2
+G_REMOTE_VERSION_RC=0
```

Comment:
Let's start with `RC=-1` here, like we did before v7, so that we have a clear development version and RC and start the first beta with `RC=0` as a clear indicator. I'll commit it.